### PR TITLE
Console ClusterRole added permissions

### PR DESCRIPTION
**Pull request by showeimer**

### CODE REVIEW

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 1]
```diff
@@ -19,35 +19,37 @@ metadata:
...
+  - console.openshift.io
+  - project.openshift.io
   resources:
-  - 'infrastructures'
-  - 'consolelinks'
-  - 'projects'
+  - infrastructures
+  - consolelinks
+  - projects
+  - featuregates
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:43]: Add FeatureGates for discovery feature

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 2]
```diff
@@ -19,35 +19,37 @@ metadata:
...
   verbs:
   - list
   - get
+  - watch
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:47]: Add the watch for FeatureGates

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 3]
```diff
@@ -56,7 +58,10 @@ rules:
   apiGroups:
   - ''
   resources:
+  - configmaps
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:61]: To enable grabbing any needed configmaps, for example OCP console's configmap for their route

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 4]
```diff
@@ -56,7 +58,10 @@ rules:
...
+  - jobs
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:62]: Might be needed for cluster curator work

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 5]
```diff
@@ -56,7 +58,10 @@ rules:
...
   - namespaces
+  - pods
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:64]: We grab the pods for provision/deprovision jobs

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 6]
```diff
@@ -79,7 +84,11 @@ rules:
   - hive.openshift.io
   resources:
   - clusterdeployments
+  - clusterpools
   - clusterimagesets
+  - clusterprovisions
+  - clusterdeprovisions
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:90]: Currently we grab the ClusterProvisions for the provision failure reason; I think there might be some information in the ClusterDeprovision we might be able to leverage

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 7]
```diff
@@ -79,7 +84,11 @@ rules:
   - hive.openshift.io
   resources:
   - clusterdeployments
+  - clusterpools
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:87]: Future UI work

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 8]
```diff
@@ -79,7 +84,11 @@ rules:
...
   - clusterimagesets
+  - clusterprovisions
+  - clusterdeprovisions
+  - machinepools
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:91]: For scaling feature

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 9]
```diff
@@ -95,6 +104,7 @@ rules:
   - cluster.open-cluster-management.io
   resources:
   - managedclusters
+  - managedclustersets
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:107]: For Future UI work

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 10]
```diff
@@ -108,4 +118,12 @@ rules:
   apiGroups:
   - discovery.open-cluster-management.io
   resources:
-  - discoveryconfigs
\ No newline at end of file
+  - discoveryconfigs
+  - discoveredclusters
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:122]: For current on going discovery work

File: stable/console-chart/templates/console-clusterrole.yaml [hunk 11]
```diff
@@ -108,4 +118,12 @@ rules:
...
+
+- verbs:
+  - watch
+  apiGroups:
+  - wgpolicyk8s.io
+  resources:
+  - policyreports
```
Comment (showeimer) [inline stable/console-chart/templates/console-clusterrole.yaml:129]: For CCX insights work

**Review by jamestalton — APPROVED** [review]
lgtm

**Pull request merged**